pi-apps-terminal-bash-edition.sh

```diff
@@ -67,76 +57,28 @@ function help() {
 	echo -e "\n${white}${inverted}${bold}${light_blue}USAGE:${normal}"
 	echo '-------'
 	#echo -e "${underline}${light_green}./pi-apps-terminal-bash-edition.sh [option]${normal}"
-	echo -e "${white}${underline}${light_green}pi-apps [option]${normal}"
+	echo -e "${white}${underline}${light_green}pi-apps [option] [args]${normal}"
 	echo -e "\n${white}${inverted}${bold}${light_blue}Available options:${normal}"
 	echo '-------------------'
-	echo -e "${white}${dark_grey_background}install [appname]${normal}${white} - install any app available in pi-apps.\n"
-	echo -e "${white}${dark_grey_background}remove [appname]${normal}${white} - uninstall any app available in pi-apps. you can also use ${dark_grey_background}uninstall${normal}.\n"
-	echo -e "${white}${dark_grey_background}multi-install [app1] [app2]${normal}${white} - install multiple apps at the same time.\n"
-	echo -e "${white}${dark_grey_background}multi-uninstall [app1] [app2]${normal}${white} - uninstall multiple apps at the same time. you can also use ${dark_grey_background}multi-remove${normal}${white}.\n"
-	echo -e "${white}${dark_grey_background}reinstall [appname]${normal}${white} - reinstall any app available in pi-apps.\n"
+	echo -e "${white}${dark_grey_background}install [appname]${normal}${white} - install any apps available in pi-apps.\n"
+	echo -e "${white}${dark_grey_background}uninstall/remove [appname]${normal}${white} - uninstall any apps available in pi-apps.\n"
+	echo -e "${white}${dark_grey_background}reinstall [appname]${normal}${white} - reinstall any apps available in pi-apps.\n"
 	echo -e "${white}${dark_grey_background}list-all${white}${normal}${white} - print all apps available in pi-apps.\n"
 	echo -e "${white}${dark_grey_background}list-installed${normal}${white} - print all installed apps.\n"
 	echo -e "${white}${dark_grey_background}list-uninstalled${normal}${white} - print all uninstalled apps.\n"
-	echo -e "${white}${dark_grey_background}list-corrupted${normal}${white} - print all apps with the corrupted statu (meaning they failed to install/uninstall).\n"
+	echo -e "${white}${dark_grey_background}list-corrupted${normal}${white} - print all apps failed to install/uninstall.\n"
```

Comment:
```suggestion
	echo -e "${white}${dark_grey_background}list-corrupted${normal}${white} - print all apps failed to install/uninstall (are corrupted).\n"
```
just to make it consistent with the GUI.